installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -70,4 +70,4 @@ include::modules/ipi-install-verifying-static-ip-address-configuration.adoc[leve
 [role="_additional-resources"]
 .Additional resources
 
-* See xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[{product-title} upgrade channels and releases] for an explanation of the different release channels.
+* See xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[{product-title} upgrade channels and releases] for an explanation of the different release channels.
```

Comment:
To align with the new *Additional resources* formatting guidelines:

```suggestion
* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[{product-title} upgrade channels and releases]
```

Reply:
Updated as suggested.